Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM tomcat:9.0-alpine
+FROM tomcat:9.0.10-jre8-alpine
```

Comment:
Why do we care about the specific Tomcat patch number?

Should we use a more recent JRE version?

Reply:
I don't think we care about this specific version, per se.  This appears to be the latest Alpine available, which is why I chose it.